src/styles/move.scss

```diff
@@ -0,0 +1,55 @@
+.move {
+
+  &__title {
+    font-size: 32px;
+    line-height: 32px;
+    text-align: left;
+    margin-top: 60px;
+    margin-bottom: 32px;
+  }
+
+  &__text {
+    font-size: 23px;
+    line-height: 32px;
+    text-align: left;
+  }
+
+}
+
+
+@media (min-width: 744px) {
+
+
+  .move {
+    &__content {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    &__title {
+      font-size: 48px;
+      margin-top: 120px;
+    }
+
+    &__text {
+      display: block;
+      width: 288px;
+      margin-top: 120px;
+    }
+  }
+
+  }
+
+@media (min-width: 1260px) {
+  .move {
+
+    &__title {
+      font-size: 64px;
+      line-height: 64px;
+    }
+
+    &__text {
+      width: 498px;
```

Comment:
use max-width better variant. 
Also using static width or height not good idea. ( if you have alternative path)